docker/build/installers/FastRTPS_1.5.0.patch

```diff
@@ -0,0 +1,1124 @@
+diff -Npur -X ex.file a/cmake/dev/check_configuration.cmake b/cmake/dev/check_configuration.cmake
```

Comment:
Patch is hard to maintain. Please make a fork of the upstream repo, submit the code into the fork, and refer to it from Apollo. So that everybody can see what is changed, and it's easy to rebase to newer version of upstream.